examples/01_dataset_simulation.ipynb

```diff
@@ -46,7 +46,7 @@
     {
```

Comment:
<p>is here still the chess board pattern for the imag image?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radionets-project/radionets/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
maybe my plots were just outdated

Reply:
<p>or I got confused because of the white noise</p>